cclib/parser/daltonparser.py

```diff
@@ -48,14 +48,13 @@ def before_parsing(self):
         # would like to use as triggers.
         self.section = None
 
-        # If there is no symmetry, assume this.
-        self.symlabels = ['Ag']
+        # If there is no symmetry (point group is C1), assume this.
```

Comment:
Could you expand this comment? It's not clear what "assume this" means for the reset of the code.